### DISCUSSION TITLE
Sync bitcoin blockchain

### DISCUSSION THREAD

**Discussion by elpro234** [ordinals/ord#1806] [Q&A]
Hi, I have a question about the syncing of bitcoin blockchain,  I think maybe the syncing will take time more than the downloading one, I did run `bitcoind -txindex ` and already waiting for 6 hours and still the progress is 0.13% knowing that I have a good settings in my computer (RAM, cpu, storage and a good WIFI)
![image](https://user-images.githubusercontent.com/116670331/220080405-b1517742-ca3a-40e4-99f3-ca902dd1bea3.png)

**Comment by tuncatunc** [comment]
You may not have enough number of peers.
Please check number of peers. I've currently have 23 peers on testnet.

`bitcoin-cli getpeerinfo`
[https://developer.bitcoin.org/reference/rpc/getpeerinfo.html](https://developer.bitcoin.org/reference/rpc/getpeerinfo.html)
